Return-Path: <alsa-devel-bounces@alsa-project.org>
X-Original-To: diwic@mail.canonical.com
Delivered-To: diwic@mail.canonical.com
Received: from fiordland.canonical.com (fiordland.canonical.com [91.189.94.145])
	by grenadilla.canonical.com (Postfix) with ESMTP id D379314720FE
	for <diwic@mail.canonical.com>; Fri, 30 Jan 2015 06:22:21 +0000 (UTC)
Received: from cluster-j.mailcontrol.com (cluster-j.mailcontrol.com [85.115.54.190])
	by fiordland.canonical.com (Postfix) with ESMTPS id C959BA180FD
	for <david.henningsson@cleanmail.canonical.com>; Fri, 30 Jan 2015 06:22:20 +0000 (UTC)
Received: from arctowski.canonical.com (arctowski.canonical.com [91.189.94.158])
	by rly47j.srv.mailcontrol.com (MailControl) with ESMTP id t0U6MJL9038885
	for <david.henningsson@cleanmail.canonical.com>; Fri, 30 Jan 2015 06:22:19 GMT
Received: from fiordland.canonical.com ([91.189.94.145])
	by arctowski.canonical.com with esmtp (Exim 4.71)
	(envelope-from <alsa-devel-bounces@alsa-project.org>)
	id 1YH4yB-0007UN-FQ
	for david.henningsson@cleanmail.canonical.com; Fri, 30 Jan 2015 06:22:19 +0000
Received: from alsa0.perex.cz (alsa0.perex.cz [77.48.224.243])
	by fiordland.canonical.com (Postfix) with ESMTP id 52E0DA180FD
	for <david.henningsson@canonical.com>; Fri, 30 Jan 2015 06:22:19 +0000 (UTC)
Received: by alsa0.perex.cz (Postfix, from userid 1000)
	id C47C62608EC; Fri, 30 Jan 2015 07:22:18 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.2.4 (2008-01-01) on mail1.perex.cz
X-Spam-Level: ****
X-Spam-Status: No, score=4.7 required=5.0 tests=DNS_FROM_AHBL_RHSBL,
	ROUND_THE_WORLD_LOCAL autolearn=disabled version=3.2.4
Received: from alsa0.perex.cz (localhost [IPv6:::1])
	by alsa0.perex.cz (Postfix) with ESMTP id 3713126067B;
	Fri, 30 Jan 2015 07:22:07 +0100 (CET)
X-Original-To: alsa-devel@alsa-project.org
Delivered-To: alsa-devel@alsa-project.org
Received: by alsa0.perex.cz (Postfix, from userid 1000)
 id C313926067B; Fri, 30 Jan 2015 07:22:04 +0100 (CET)
Received: from rtits2.realtek.com.tw (rtits2.realtek.com [60.250.210.242])
 by alsa0.perex.cz (Postfix) with ESMTP id 75F5A260676
 for <alsa-devel@alsa-project.org>; Fri, 30 Jan 2015 07:21:57 +0100 (CET)
Authenticated-By: 
X-SpamFilter-By: BOX Solutions SpamTrap 5.52 with qID t0U6LlmE005850,
 This message is accepted by code: ctloc85258
Received: from mail.realtek.com (rtitcas11.realtek.com.tw[172.21.6.12])
 by rtits2.realtek.com.tw (8.14.9/2.40/5.64) with ESMTP id t0U6LlmE005850
 (version=TLSv1/SSLv3 cipher=AES128-SHA bits=128 verify=NOT);
 Fri, 30 Jan 2015 14:21:50 +0800
Received: from sw-server.rtdomain (172.21.81.164) by RTITCAS11.realtek.com.tw
 (172.21.6.12) with Microsoft SMTP Server id 14.3.210.2;
 Fri, 30 Jan 2015 14:21:48 +0800
From: Bard Liao <bardliao@realtek.com>
To: <broonie@kernel.org>, <lgirdwood@gmail.com>
Date: Fri, 30 Jan 2015 14:21:33 +0800
Message-ID: <1422598893-28393-1-git-send-email-bardliao@realtek.com>
X-Mailer: git-send-email 1.8.1.1.439.g50a6b54
MIME-Version: 1.0
X-Originating-IP: [172.21.81.164]
Cc: oder_chiou@realtek.com, Bard Liao <bardliao@realtek.com>,
        alsa-devel@alsa-project.org, flove@realtek.com,
        liam.r.girdwood@intel.com
Subject: [alsa-devel] [PATCH] ASoC: rt286: Add rt288 codec support
X-BeenThere: alsa-devel@alsa-project.org
X-Mailman-Version: 2.1.14
Precedence: list
List-Id: "Alsa-devel mailing list for ALSA developers -
 http://www.alsa-project.org" <alsa-devel.alsa-project.org>
List-Unsubscribe: <http://mailman.alsa-project.org/mailman/options/alsa-devel>, 
 <mailto:alsa-devel-request@alsa-project.org?subject=unsubscribe>
List-Archive: <http://mailman.alsa-project.org/pipermail/alsa-devel/>
List-Post: <mailto:alsa-devel@alsa-project.org>
List-Help: <mailto:alsa-devel-request@alsa-project.org?subject=help>
List-Subscribe: <http://mailman.alsa-project.org/mailman/listinfo/alsa-devel>, 
 <mailto:alsa-devel-request@alsa-project.org?subject=subscribe>
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Errors-To: alsa-devel-bounces@alsa-project.org
Sender: alsa-devel-bounces@alsa-project.org
X-Mailcontrol-Inbound: WBq!ucXEz80wFmxD6S7f3nyjenN!phNbIQWiyBG44+lo3FL0yiLKb3yjenN!phNbh+9WRKrdgbE=
X-Spam-Score: -0.3
X-Scanned-By: MailControl 37812.332 (www.mailcontrol.com) on 10.74.0.157

This patch adds support for rt288 codec.

Signed-off-by: Bard Liao <bardliao@realtek.com>
---
 sound/soc/codecs/rt286.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rt286.c b/sound/soc/codecs/rt286.c
index 81bdd276..2eafbcf 100644
--- a/sound/soc/codecs/rt286.c
+++ b/sound/soc/codecs/rt286.c
@@ -34,6 +34,7 @@
 #include "rt286.h"
 
 #define RT286_VENDOR_ID 0x10ec0286
+#define RT288_VENDOR_ID 0x10ec0288
 
 struct rt286_priv {
 	struct regmap *regmap;
@@ -1210,7 +1211,7 @@ static int rt286_i2c_probe(struct i2c_client *i2c,
 
 	regmap_read(rt286->regmap,
 		RT286_GET_PARAM(AC_NODE_ROOT, AC_PAR_VENDOR_ID), &ret);
-	if (ret != RT286_VENDOR_ID) {
+	if (ret != RT286_VENDOR_ID && ret != RT288_VENDOR_ID) {
 		dev_err(&i2c->dev,
 			"Device with ID register %x is not rt286\n", ret);
 		return -ENODEV;
-- 
1.8.1.1.439.g50a6b54

_______________________________________________
Alsa-devel mailing list
Alsa-devel@alsa-project.org
http://mailman.alsa-project.org/mailman/listinfo/alsa-devel

